src/index.css

```diff
@@ -1,3 +1,18 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+
+.css-dip3t8 {
+    position: relative;
+    height: 100%;
+    overflow-y: auto;
+    overflow-x: hidden;
+    z-index: 3;
+    background-color: inherit !important;
+}
+
+.css-1t8x7v1 >.ps-menu-button:hover {
+    background-color: #164d41 !important;
```

Comment:
We need to avoid having fixed colors and use css variables instead, as we will need to implement light/dark mode feature. Having fixed colors will make this very difficult to implement.